app/views/claims/schools/claims/show.html.erb

```diff
@@ -18,7 +18,7 @@
 
       <% if policy(@claim).submit? %>
         <%= govuk_button_to t(".submit"), check_claims_school_claim_path(@school, @claim), method: :get %>
-      <% elsif @claim.submitted? %>
+      <% elsif @claim.submitted_by_id? %>
```

Comment:
This was previously checking the status. All statuses past the `submitted` status should have the `submitted_by_id` and `submitted_at` values.